src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/CourtValidationService.kt

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.service
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.CourtId
+import uk.gov.justice.digital.hmpps.managerecallsapi.register.court.CourtRegisterClient
+
+@Service
+class CourtValidationService(@Autowired private val courtRegisterClient: CourtRegisterClient) {
+
+  fun isValid(courtId: CourtId?): Boolean {
+    return courtId?.let {
+      courtRegisterClient.getAllCourts().block()?.any { (it.courtId == courtId) } ?: false
```

Comment:
should probably do the same as the prison validation and use the get by id endpoint